### PR TITLE
Update overview messaging and structure

### PR DESCRIPTION
**Pull request by pavithraes**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### Any other comments?

Didn't realize we already had an assets folder, so I've removed the images folder created in the previous PR!

### CODE REVIEW

File: 00_overview.ipynb [hunk 1]
```diff
@@ -149,7 +170,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.6"
+   "version": "3.9.13"
```
Comment (pavithraes) [inline 00_overview.ipynb:173]: Oh, I'll update my local env!
```suggestion
   "version": "3.10.6"
```

**Review by iameskild — APPROVED** [review]
Thanks for the updates :)

File: 00_overview.ipynb [hunk 2]
```diff
@@ -7,131 +7,205 @@
...
-    "<h1 style=\"text-align: center;\">Welcome to the Nebari Tutorial!</h1>\n",
-    "<h2 style=\"text-align: center;\">PyDataNYC - 2022</h2>\n",
-    "\n",
-    "\n",
-    "![logo-mark](https://raw.githubusercontent.com/nebari-dev/nebari-design/main/logo-mark/colored-background/Nebari-Logo-Black-Bg.jpg)"
+    "<div align=\"center\">\n",
+    "    <h1>Welcome to the Nebari tutorial!</h1>\n",
+    "    <h2> PyData NYC, 2022</h2>\n",
+    "    <hr>\n",
+    "    <img src=\"../assets/nebari-banner.svg\" alt=\"Nebari logo mark with the tagline Your open source data science platform: Built for scale, designed for collaboration.\" width=\"70%\">\n",
```
Comment (iameskild) [inline 00_overview.ipynb:14]:
```suggestion
    "    <img src=\"./assets/nebari-banner.svg\" alt=\"Nebari logo mark with the tagline Your open source data science platform: Built for scale, designed for collaboration.\" width=\"70%\">\n",
```

**Review by iameskild — APPROVED** [review]
Minor fix to render image.

**Pull request merged**